frontend/src/app/pages/registration/registration.component.ts

```diff
@@ -46,7 +47,8 @@ export class RegistrationComponent implements OnInit, OnDestroy {
         Validators.pattern(this.containsNumberRegex),
         Validators.pattern(this.constainsSpecialCharacterRegex)
       ]),
-      confirmPassword: new FormControl('', [Validators.required])
+      confirmPassword: new FormControl('', [Validators.required]),
+      userType: new FormControl<boolean>(false),
```

Comment:
what about image? what will happen when no image is provided and someone wants to access it later? Some default image? I guess the easiest solution is just to make it required

Reply:
think its weird to force users to upload an image

Reply:
No way we have that in the specification